platform/xbox/functions/time/time.c

```diff
@@ -9,13 +9,15 @@
 #ifndef REGTEST
 
 #include <xboxkrnl/xboxkrnl.h>
+#define NT_EPOCH_TIME_OFFSET ((LONGLONG)(369 * 365 + 89) * 24 * 3600 * 10000000)
```

Comment:
I'm surprised this works (does it?). `3600 * 10000000` is more than 32-bit. I'd assume this has to be `3600LL * 10000000LL` etc. (I keep forgetting about C int promotion rules, so maybe it's already fine?)

Reply:
It works. That's from ntfs-3g ntfstime.h. The compiler uses the first type that fits. Better explained there: https://stackoverflow.com/questions/13134956/what-is-the-reason-for-explicitly-declaring-l-or-ul-for-long-values.